crates/router/src/core/payments/helpers.rs

```diff
@@ -2037,6 +2037,23 @@ pub fn decide_payment_method_retrieval_action(
     }
 }
 
+pub async fn is_ucs_enabled(state: &SessionState, config_key: &str) -> RouterResult<bool> {
+    let db = state.store.as_ref();
+    match db.find_config_by_key(config_key).await {
```

Comment:
@sai-harsha-vardhan Should this have been `find_config_by_key_unwrap_or()` instead, or is the current implementation correct?